scalar/time.go

```diff
@@ -94,3 +94,7 @@ func (s *Time) Set(value any) error {
 		return s.Int.Set(value)
 	}
 }
+
+var (
+	_ Scalar = (*Time)(nil)
+)
```

Comment:
Not adding ByteSize() to Time scalars? Is this on purpose?

Reply:
yes, as they really occupy only the underlying Int, which we get from embedding